hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.js

```diff
@@ -103,4 +98,60 @@ const Navbar = () => (
     </nav>
 );
 
-export default Navbar;
+export const NavbarDrawer = () => (
+    <nav className={styles.navMobile}>
+        <div>
+            <Link to={"/"}>
```

Comment:
Are all of these routes the same here an in the regular `<Navbar>`, and will that always be the case?

Seems uncessary to me to duplicate all this code if so. You could pull this out to it's own component, with a bit of logic to determine whether `styles.navBtn` or `styles.navMobileBtn` gets applied. Eg
```jsx
const btnClass = mobile ? styles.navMobileBtn : styles.navBtn 
```